src/cms/templates/pages/page_tree.html

```diff
@@ -81,8 +81,8 @@ <h2 class="heading font-normal mb-2">{% trans 'Pages' %}</h2>
                 <th class="text-sm text-left uppercase py-3 px-2">
                     <div class="lang-grid flags whitespace-nowrap">
                         {% for lang in languages %}
-                            <a href="{% url 'pages' region_slug=region.slug language_code=lang.code %}">
-                                <span title="{{ lang.translated_name }}"><img src="{% static '' %}images/flags/{{ lang.code }}.png" /></span>
+                            <a href="{% url 'pages' region_slug=region.slug language_slug=lang.slug %}">
+                                <span title="{{ lang.translated_name }}"><img src="{% static '' %}images/flags/{{ lang.slug }}.png" /></span>
```

Comment:
Since the url of these images changed, we should probably also rename the files in the [flags directory](https://github.com/Integreat/integreat-cms/tree/develop/src/cms/static/images/flags)...

![Screenshot_2021-03-04 Integreat Redaktionssystem](https://user-images.githubusercontent.com/16021269/110051807-cd4d8080-7d56-11eb-8045-1c6f88085710.png)

Reply:
Oh, good point. As the slug is unique, it probably works anyways.